includes/Checker/CLI_Runner.php

```diff
@@ -57,8 +57,19 @@ public static function is_plugin_check() {
 	 * @throws Exception Thrown if the plugin parameter is empty.
 	 */
 	protected function get_plugin_param() {
-		// Get the plugin parameter from the command line arguments.
-		$plugin = isset( $_SERVER['argv'][3] ) ? $_SERVER['argv'][3] : '';
+		// Exclude first three reserved elements.
+		$params = array_slice( $_SERVER['argv'], 3 );
+
+		// Remove associative arguments.
+		$params = array_filter(
+			$params,
+			function ( $val ) {
```

Comment:
```suggestion
			static function ( $val ) {
```

Per [729](https://github.com/WordPress/performance/pull/729), Use static closures for minor performance improvement whenever instance access is not needed.